content/work/moztrap.rst

```diff
@@ -18,10 +18,10 @@ tasks:
   - Project Architecture
   - Responsive Styles
 quotes:
-  - text: 'I consider MozTrap to be one of the best
+  - text: 'I consider <a href="/work/">Mozilla’s MozTrap</a> to be one of the best
```

Comment:
That's true. We'll link to it as soon as we have a case study. That's soon on the list...